services.py

```
@@ -500,50 +492,81 @@ def _suggest_next_steps(scores: Dict[str, float]) -> List[str]:
+                UserAchievement.achievement_id.in_(achievement_ids)
+            ).all()
+            
+            # 既存アチーブメントをIDでマッピング
+            existing_map = {ua.achievement_id: ua for ua in existing_user_achievements}
+            
+            # 新規追加用のリスト
+            new_achievements = []
+            
+            # アチーブメントごとにチェック
+            for achievement in achievements:
+                score = skill_achievements.get(achievement.threshold_type, 0)
```

Ruff (0.12.2)

Local variable `score` is assigned to but never used

Remove assignment to unused variable `score`

(F841)